### DISCUSSION TITLE
macOS-latest updated to macOS-12

### DISCUSSION THREAD

**Discussion by Allje** [actions/virtual-environments#5640] [Image configuration]
Hi

Is the a plan for when "macOS-latest" will be updated to "macOS-12" ? At the moment "macOS-latest" is set to "macOS-11"

**Comment by al-cheb** [comment]
Yep, we have  - https://github.com/actions/virtual-environments/issues/5583#issuecomment-1134941811

**Reply by Allje** [reply]
Thanks 👍

**Reply by sandyarmstrong** [reply]
https://github.com/actions/runner-images/pull/5594 merged. What is preventing pointing `macos-latest` to `macos-12` now?

**Reply by mikhailkoliada** [reply]
@sandyarmstrong we have plans on it, it will be announced a bit later.
